packages/synapse-react-client/src/components/CreateOrUpdateAccessRequirementWizard/CreateOrUpdateAccessRequirementWizard.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import { Meta, StoryObj } from '@storybook/react'
+import { fn } from '@storybook/test'
+import CreateOrUpdateAccessRequirementWizard from './CreateOrUpdateAccessRequirementWizard'
+import { RestrictableObjectType } from '@sage-bionetworks/synapse-types'
+import {
+  mockACTAccessRequirement,
+  mockToUAccessRequirement,
+} from '../../mocks/mockAccessRequirements'
+
+const meta = {
+  title: 'Governance/CreateOrUpdateAccessRequirementWizard',
+  component: CreateOrUpdateAccessRequirementWizard,
+  parameters: {
+    stack: 'development',
+  },
+} satisfies Meta
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const CreateTeamAR: Story = {
+  args: {
+    open: true,
+    onComplete: fn(),
+    onCancel: fn(),
+    subject: {
+      id: '3429759',
+      type: RestrictableObjectType.TEAM,
+    },
+  },
+}
+
+export const CreateEntityAR: Story = {
+  args: {
+    open: true,
+    onComplete: fn(),
+    onCancel: fn(),
+    subject: {
+      id: 'syn12714410',
+      type: RestrictableObjectType.ENTITY,
+    },
+  },
+}
+
+export const EditManagedAR: Story = {
+  args: {
+    open: true,
+    onComplete: fn(),
+    onCancel: fn(),
+    accessRequirementId: '9602671',
+  },
+}
+
+export const EditSelfSignAR: Story = {
+  args: {
+    open: true,
+    onComplete: fn(),
+    onCancel: fn(),
+    accessRequirementId: '9602700',
+  },
+}
+
+/* For Mock Stories, updates to Wiki Instructions will not be visible until MarkdownSynapse is updated to use react-query. */
```

Comment:
When the user updates the wiki content via `useUpdateWikiPage`, react query will update the cache for that wiki page. If a component gets a wiki page via `useGetWikiPage`, then the updated wiki page content will be returned, even when using mocks. However, MarkdownSynapse calls `SynapseClient.getEntityWiki` rather than using the react-query cache. So in mock stories, the MSW handler will intercept the GET request and return the original wiki page, rather than the updated page, and the wiki instructions will appear not to update after editing. 

Since stories against the dev stack will show the expected behavior, I held off on attempting to update our MSW handlers to accommodate this case.

Reply:
Should MarkdownSynapse be changed to use useGetWikiPage?  Is the problem that MarkdownSynapse a an old class-based component?

Reply:
Yes and yes! Since MarkdownSynapse is a class-based component, I was concerned that the refactor to use react-query could be complicated. Should I open a new ticket to capture the refactor work?  We could also tackle the refactor as part of https://sagebionetworks.jira.com/browse/SWC-6791